R/pagination.R

```diff
@@ -903,49 +1027,30 @@ paginate_indices <- function(obj,
     )
   }
 
-  ## I'm not sure this is worth doing.
-  ## ## We can't support forced pagination here, but we can support calls to,
-  ## ## e.g., paginate_indices(do_forced_pag(tt))
-  ## if(is.list(obj) && !is.object(obj)) {
-  ##     res <- lapply(obj, paginate_indices,
-  ##                   page_type = page_type,
-  ##                   font_family = font_family,
-  ##                   font_size = font_size,
-  ##                   lineheight = lineheight,
-  ##                   landscape = landscape,
-  ##                   pg_width = pg_width,
-  ##                   pg_height = pg_height,
-  ##                   margins = margins,
-  ##                   lpp = lpp,
-  ##                   cpp = cpp,
-  ##                   tf_wrap = tf_wrap,
-  ##                   max_width = max_width,
-  ##                   colwidths = colwidths,
-  ##                   min_siblings = min_siblings,
-  ##                   nosplitin = nosplitin,
-  ##                   col_gap = col_gap,
-  ##                   ## not setting num_rep_cols here cause it wont' get it right
-  ##                   verbose = verbose)
-  ##     return(splice_idx_lists(res))
-  ## }
```

Comment:
I think we can remove this commented code here

Reply:
yeah that code has been commented out for a year, which is why I removed it while I was in the function anyway...